action.py

```diff
@@ -72,6 +72,10 @@ def _fatal_help(msg):
 if _DEBUG:
     sigstore_python_env["SIGSTORE_LOGLEVEL"] = "DEBUG"
 
+identity_token = os.getenv("GHA_SIGSTORE_PYTHON_IDENTITY_TOKEN")
+if identity_token != "":
+    sigstore_sign_args.extend(["--identity_token", identity_token])
```

Comment:
I think this should be:

```suggestion
    sigstore_sign_args.extend(["--identity-token", identity_token])
```

Reply:
Good catch, I was about to test that in #23 haha.